endpoints/openrtb2/auction.go

```diff
@@ -1582,44 +1583,76 @@ func getJsonSyntaxError(testJSON []byte) (bool, string) {
 // Note that processStoredAuctionResponses must be called after processStoredRequests
 // because stored imps and stored requests can contain stored auction responses
 // so the stored requests/imps have to be merged into the incoming request prior to processing stored auction responses.
-func (deps *endpointDeps) processStoredAuctionResponses(ctx context.Context, requestJson []byte) (map[string]json.RawMessage, []error) {
+func (deps *endpointDeps) processStoredAuctionResponses(ctx context.Context, requestJson []byte) (map[string]json.RawMessage, map[string]map[string]json.RawMessage, []error) {
```

Comment:
This function extracts storedResponses and storedBidResponses from raw request. We use the same fetcher for both of them. Because of parsing a raw request we extract all ids and return them `return impIdToStoredResp, impBidderToStoredBidResponse, nil`. What do you think would be a good split?